assets/app/data/crowbar/services/crowbar-backup.factory.js

```diff
@@ -23,9 +23,9 @@
 
             var requestOptions = {
                 method: 'POST',
-                url: '/utils/backups',
```

Comment:
the data services should be split by the first url component (after `api`) so this whole function should be moved to `upgradeFactory`.